internal/osutils/autostart/autostart_linux.go

```diff
@@ -81,12 +82,18 @@ func (a *app) enableOnServer() error {
 	}
 
 	esc := osutils.NewBashEscaper()
-	exec := esc.Quote(a.Exec)
+	// Avoid output to the console when starting up
+	exec := fmt.Sprintf("%s > /dev/null 2>&1", a.Exec)
```

Comment:
This is the wrong place to address showing the title IMO. If there are real errors such as the current one I wouldn't mind that it is explicit and end-users thus have something to report to us.

Instead I think we can just drop printing titles from the state-svc altogether. It's not meant to be run by users manually anyway.